### PR TITLE
refactor: add minimum of minFeeBroadcast for getFeeStatistics query

### LINKED ISSUE #1741: Default feestatistics should match default node settings
**Is your feature request related to a problem? Please describe.**
[getFeestatistics()](https://github.com/ArkEcosystem/core/blob/fad5a259b1b1c074e7cf35d8279371ac78a47062/packages/core-graphql/src/repositories/transactions.ts#L257) should have a minimum for the query for fees. This would reflect a non-adjusted node's fee structure with offset. Currently this includes delegates with modified fee structures self-packaging transactions in their blocks, which drags the minimum down and probably the average. The current model reflects on delegates as they have a very high throughput on the network, however only they are able to make these transactions. I'm not sure how feestatistics are useful for delegates, because of the aforementioned method.

### PR DESCRIPTION
**Pull request by vulet**
## Proposed changes 
Resolves #1741.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [ ] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/core-graphql/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "dependencies": {
         "@arkecosystem/core-container": "^2.1.0",
         "@arkecosystem/core-http-utils": "^2.1.0",
+        "@arkecosystem/core-transaction-pool": "^2.1.0",
```
Comment (faustbrian) [inline packages/core-graphql/package.json]: Packages itself should never have a dependency on anything but interfaces and the container.

File: packages/core-api/src/repositories/transactions.ts [hunk 2]
```diff
@@ -248,6 +249,7 @@ export class TransactionsRepository extends Repository implements IRepository {
      * @return {Object}
      */
     public async getFeeStatistics(): Promise<any> {
+        const minFeeBroadcast = localConfig.get("dynamicFees.minFeeBroadcast");
```
Comment (faustbrian) [inline packages/core-api/src/repositories/transactions.ts]: This should be resolved from the container, those config objects are only for plugin internal use.

File: packages/core-graphql/src/repositories/transactions.ts [hunk 3]
```diff
@@ -255,6 +256,7 @@ class TransactionsRepository extends Repository {
      * @return {Object}
      */
     public async getFeeStatistics() {
+        const minFeeBroadcast = localConfig.get("dynamicFees.minFeeBroadcast");
```
Comment (faustbrian) [inline packages/core-graphql/src/repositories/transactions.ts]: This should be resolved from the container, those config objects are only for plugin internal use.

**Comment by vulet** [comment]
I see, something like this?

**Comment by boldninja** [comment]
@vulet Can you provide me with your ARK address for GitHub bounty? You can also contact me on ARK's Slack - https://ark.io/slack (username : boldninja) and direct message me your address there.

**Pull request merged**